src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
```

Comment:
Fair point. Wanna mull on this some more though... feels a little like the moment we need any additional small dependency in future, we'd have to make a backwards incompatible change to this driver.